fdbserver/TLogInterface.h

```diff
@@ -255,4 +261,53 @@ struct TLogQueuingMetricsRequest {
 	}
 };
 
+struct TLogDisablePopRequest {
+	constexpr static FileIdentifier file_identifier = 4022806;
+	Arena arena;
+	UID snapUID;
+	ReplyPromise<Void> reply;
+	Optional<UID> debugID;
+
+	TLogDisablePopRequest() {}
```

Comment:
```suggestion
	TLogDisablePopRequest() = default;
```

And similarly for the ones below.